Jenkinsfile

```diff
@@ -37,12 +43,18 @@ pipeline {
                     buildEnd=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
                     imagesha=$(cat image | tr -d '[:space:]')
                     tag=$(cat image-tag)
+                    accessToken=$(cat access-token | tr -d '[:space:]' | tr -d '\n')
```

Comment:
We might want to put `set +x` at the start of this block `sh` block, otherwise the access token is all over the build logs because Jenkins sets debug mode on all scripts. Or mask the access token some other way, although it looks like that can only be done for static secrets.